### ISSUE TITLE
[Docs] No documentation mentions the ability to host bot logic in browser

### ISSUE THREAD

**Issue by sgellock** [microsoft/BotBuilder-Samples#893]
## Describe the bug
Now that there are effectively two ways to host a bot (through the bot service and in-browser) deployment documentation should shed light on the in-browser scenario. 

[bug]

**Comment by Kaiqb** [comment]
The sample that shows how to host bot in the browser is broken, so this work will need to be pushed out.

**Comment by CoHealer** [comment]
@sgellock, @EricDahlvang said it works in Chrome but seeing an error in Edge:

SCRIPT1028: SCRIPT1028: Expected identifier, string or number
app.js (16292,24)

Thinking it might be the spread operator. Assigning to @anusharavindrar  to investigate and fix.

**Comment by CoHealer** [comment]
@anusharavindrar, please comment on your findings.

**Comment by anushaRaoR** [comment]
- Tried using @babel/plugin-transform-spread and then including @babel/plugin-transform-spread and @babel/plugin-proposal-object-rest-spread in the presets in .babelrc file, but it still shows the same error.
- The targets for the presets in .babelrc file only specifies for the Chrome version but not for Edge.

**Comment by anushaRaoR** [comment]
Fix:https://github.com/microsoft/botbuilder-js/pull/1056

**Comment by anushaRaoR** [comment]
Fix: https://github.com/microsoft/botbuilder-js/pull/1056

**Comment by sgellock** [comment]
@anushaRaoR I don't think we're going to take that fix as a different solution is in order.  We are working through that alternate solution.

**Comment by anushaRaoR** [comment]
@sgellock sure, would wait for the alternate solution and update the fix link accordingly

**Comment by CoHealer** [comment]
@sgellock , wasn't this fixed with the 4.5 roll out?

**Comment by sgellock** [comment]
yes, I believe so.  would be good to double check that it's fixed so the customer can get a definitive ACK from us.  Can you do that please?

**Comment by tdurnford** [comment]
In BotBuilder v4.5.3, TurnContext's `removeMentionText` method uses a lookbehind regex expression which weren't introduced until ES2018. Unfortunately, Chrome is the only browser that supports ES2018, so the sample currently  throws errors in other browsers like Edge, Firefox, and Safari. Looking at the BotBuilder source code, the Lookbehind expression that was causing the sample to fail was removed in https://github.com/microsoft/botbuilder-js/pull/1088. I tested the sample using the latest build on MyGet, and everything worked as expected. This should be resolved after the v4.6 release.
